.github/workflows/tacos_conflict.yml

```diff
@@ -38,6 +34,13 @@ jobs:
       pull-requests: write
 
     steps:
+      - name: Get mergeable state
+        env:
+          GH_TOKEN: ${{ github.token }}
+        run: |
+          echo 'STATUS='$(gh pr view ${{github.event.pull_request.html_url }} \
```

Comment:
Please either lowercase this or put a namespace on the front:

1. status -- denotes a short-lifetime env var
2. GITHUB_PR_STATUS

Reply:
changed to GITHUB_PR_STATUS.